### PR TITLE
feat: experimentally add more language support 

### PR DESCRIPTION
**Pull request by PeterDaveHello**
Added more language, including:

  - French
  - German
  - Japanese
  - Russian

It's translated by ChatGPT, of course!

May be useful, may be fun, who knows?

### CODE REVIEW

File: src/locales/ru-RU.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    yes: 'Да',
+    no: 'Нет',
+    confirm: 'Подтвердить',
+    download: 'Загрузить',
+    noData: 'Нет данных',
+    wrong: 'Произошла ошибка, попробуйте позже',
+    success: 'Успех',
+    failed: 'Неудача',
+    verify: 'Проверить',
+    unauthorizedTips: 'Не авторизован, сначала необходимо пройти проверку подлинности',
```
Comment (rzarvirov) [inline src/locales/ru-RU.ts:28]:
```suggestion
    unauthorizedTips: 'Не авторизован, сначала необходимо пройти проверку',
```

Comment (PeterDaveHello) [reply src/locales/ru-RU.ts:28]: Thanks!

**Review by 53jk1 — APPROVED** [review]

**Comment by PeterDaveHello** [comment]
@Chanzhaoyu what do you think 😄

**Comment by Chanzhaoyu** [comment]
One disadvantage is that I don't understand these languages,  Machine translation seems unreliable.

**Comment by PeterDaveHello** [comment]
OpenAI translate quality is somehow acceptable IMO, and friends come over the world can help review, the most important thing might be that: we don't really need a perfect translation or code to make it available? 😄

**Comment by Chanzhaoyu** [comment]
> 

It will also add to the trouble of the contributing staff, let me think about it. 🤔

**Comment by Chanzhaoyu** [comment]
Temporarily closed, if needed, I will contact you again, thank you~

**Pull request closed without merging**